moz_kinto_publisher/main.py

```diff
@@ -373,15 +320,13 @@ def _upload_pem(self, *, rw_client=None, kinto_id=None):
         rw_client.attach_file(
             collection=settings.KINTO_INTERMEDIATES_COLLECTION,
             fileContents=self.pemData,
-            fileName=f"{base64.urlsafe_b64encode(self.pubKeyHash).decode('utf-8')}.pem",
+            fileName=f"{base64.urlsafe_b64encode(self.derHash).decode('utf-8')}.pem",
```

Comment:
I'm a bit concerned that this will update all existing records, but maybe that's what we want anyway? (to apply this change to current records)

Reply:
Ok - I think it would be best to avoid that, so I reworked it to keep `unique_id()` how it was (and `fileName` here). How's this look?